RIOT-OS/RIOT
# docker: pass CONTINUE_ON_EXPECTED_ERRORS into container

### Contribution description

Just one more environment variable we need to preserve.

### Testing procedure

* `BOARD=arduino-mega2560 make -C examples/saul all BUILD_IN_DOCKER=1 TOOLCHAIN=llvm CONTINUE_ON_EXPECTED_ERRORS=1`

Without this patch, "The selected TOOLCHAIN=llvm is not supported." is shown twice (once outside docker, once inside), and the second one still stops compilation.

With this patch, it's still shown twice, but now "EXPECT ERRORS" is printed below *both* of them, and building is run to the compiler errors that I was hoping to see.

### Issues/PRs references

The test case happens to be exactly the invocation with which I'd test #16919, which'd fix the compiler errors.
